### DISCUSSION TITLE
Support for freepbx 17?

### DISCUSSION THREAD

**Discussion by Nelnamara** [marcelloceschia/chan-sccp-b#613] [General]
Has done any testing with this on freepbx 17 under Debain?  I have been running sccp on the distro for a few years now and would like to continue using it a bit longer.  The current release of Debian has no asterisk-dev available so I am wondering if anyone has been able to get a working build.  This may ultimately drive me to fully convert to SIP and buy a few new phones...

**Comment by PipoCanaja** [comment]
You should probably try using freepbx sources directly, without any packaging. Works without problem with asterisk at least.
